### PR TITLE
Changed default names of dropdowns

### PR DESCRIPTION
**Pull request by prerit2010**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -172,11 +172,17 @@ def data_view():
...
     ]
 
-    return render_template('index.html', response=response, workshops=workshops, show_all=True)
+    return render_template('index.html', response=response, workshops=workshops,
+                            workshop_name="All workshops", show_all=True)
 
 
 @application.route('/view/<workshop_id>/')
 def data_view_by_workshop(workshop_id):
+    """
```
Comment (rgaiacs) [inline app/views.py:178]: This isn't the docstring of the function. Please use `#`.

File: app/templates/base.html [hunk 2]
```diff
@@ -8,8 +8,9 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='css/styles.css') }}">
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
     <script src="{{ url_for('static', filename='js/plot.js') }}"></script>
+    <script src="{{ url_for('static', filename='js/call.js') }}"></script>
   </HEAD>
```
Comment (rgaiacs) [inline app/templates/base.html:12]: Why some tags use upper case letters and others lower case letters?

**Comment by rgaiacs** [comment]
@prerit2010 Looks good. Please merge.

File: app/templates/base.html (hunk 2 above)
Comment (prerit2010) [inline app/templates/base.html:12]: Will improve all of that :+1:

**Pull request merged**